### DISCUSSION TITLE
breakpoint question

### DISCUSSION THREAD

**Discussion by benjamin-cash** [NOAA-EMC/global-workflow#3085] [Q&A] [answered]
Hello - As a possible fix to this issue (https://github.com/NOAA-EMC/global-workflow/discussions/3083) I am trying a run using breakpoints and am wondering if I am using them correctly. The relevant settings are:

```
export FHMAX_GFS="4416"
# Intermediate times to stop forecast when running in segments
breakpnts="1472"
```
breakpnts is exactly one third of FHMAX_GFS, and my expectation was this would result in three 1472h segments. But looking in the experiment .xml, I have:

```
<metatask name="fcst_mem000" mode="serial">

        <var name="seg">0 1</var>
```
 Shouldn't there be three segments? And if not, how will the total run length of 4476 break down? Thanks!

**Comment by WalterKolczynski-NOAA** [comment, marked as answer]
If you only put in one breakpoint, the forecast only stops once. Here it would be 0–1472, then 1472–4476.

If you want three segments, you need two breakpoints:
```
breakpnts="1472,2944"

**Reply by benjamin-cash** [reply]
Ah, thanks! I thought that the breakpoint was every $breakpnt steps, not a list of breakpoints. Since my first segment is now running, if I edit my .xml file to add in another segment and modify breakpnts in config.base will the experiment pick that up? Or are those values used in other places as well?

**Reply by WalterKolczynski-NOAA** [reply]
I think that will work, but no guarantees.
